samples/snippets/linear_regression_tutorial_test.py

```diff
@@ -37,6 +37,20 @@ def test_linear_regression(random_model_id: str) -> None:
         replace=True,
     )
     # [END bigquery_dataframes_bqml_linear_regression]
+    # [START bigquery_dataframes_bqml_linear_evaluate]
+    import bigframes.pandas as bpd
+
+    # Select the model you will be evaluating. `read_gbq_model` loads model data from
+    # BigQuery, but you could also use the `model` object from the previous steps.
+    model = bpd.read_gbq_model(
+        your_model_id,  # For example: "bqml_tutorial.penguins_model"
+    )
+
+    # Score the model with input data defined in an earlier step to compare
+    # model predictions on feature_columns to true labels in label_columns.
+    score = model.score(feature_columns, label_columns)
```

Comment:
Would it be possible to have a comment showing what `score` looks like?

See these kinds of comments:

https://github.com/googleapis/python-bigquery-dataframes/blob/d73fe9d5fd2907aeaaa892a329221c10bb390da0/samples/snippets/create_kmeans_model_test.py#L104-L109